### PR TITLE
Cascade Remove Set Foreign Key to Null Fails When Inverse Side Is Primary

### PR DESCRIPTION
**Pull request by kristophjunge**
See #1761

### CODE REVIEW

File: src/persistence/subject-builder/OneToManySubjectBuilder.ts [hunk 1]
```diff
@@ -165,13 +165,22 @@ export class OneToManySubjectBuilder {
...
                     metadata: relation.inverseEntityMetadata,
                     parentSubject: subject,
-                    canBeUpdated: true,
                     identifier: removedRelatedEntityRelationId,
-                    changeMaps: [{
+                });
+
+                // in case the inverse relation is primary we cannot remove the relation by setting the
+                // foreign key field to null. Instead remove the entity.
+                if (relation.inverseRelation
```
Comment (pleerock) [inline src/persistence/subject-builder/OneToManySubjectBuilder.ts]: WE CANNOT SIMPLY REMOVE INVERSE ENTITY IF CASCADE WERE NOT DEFINED. Where cascade remove check is?

**Comment by pleerock** [comment]
Cascade removes is a complex topic. If you want this to be merged then we'll need to implement all cascade remove features. This PR currently covers only one to many case. We'll need to implement this for other relations as well.

File: src/persistence/subject-builder/OneToManySubjectBuilder.ts (hunk 1 above)
Comment (pleerock) [reply src/persistence/subject-builder/OneToManySubjectBuilder.ts]: also, this case only covers when foreign key of entity is a primary key, and does not cover simple cases. Cascade remove logic should not be tight to the fact of foreign key column being primary or not.

**Comment by pleerock** [comment]
Any updates?

**Comment by leandromsa** [comment]
hello, exist some solution while there is no fix realeased?

**Comment by Kononnable** [comment]
Closing as no response from the author.

**Pull request closed without merging**